lib/generamba/cli/gen_command.rb

```diff
@@ -58,5 +61,24 @@ def gen(module_name, template_name)
       generator.generate_module(module_name, code_module, template)
     end
 
+    no_commands {
+      def print_parameters_in_table(parameters, module_name)
+        params = parameters.clone
+
+        templates = []
+
+        params["templates"].each do |param|
```

Comment:
You propose introduce common class `TableParametersFormatter` or to each his own, i.e now we introduce this class only for `gen` command?